src/System.Globalization.Extensions/tests/Normalization/NormalizationAll.cs

```diff
@@ -17,45 +17,51 @@ private string ConvertToString(string codepoints)
             string[] parts = codepoints.Split('-');
             foreach (string part in parts)
             {
-                sb.Append((char) Int32.Parse(part, NumberStyles.HexNumber | NumberStyles.AllowTrailingWhite | NumberStyles.AllowLeadingWhite, CultureInfo.InvariantCulture));
+                sb.Append((char)Int32.Parse(part, NumberStyles.HexNumber | NumberStyles.AllowTrailingWhite | NumberStyles.AllowLeadingWhite, CultureInfo.InvariantCulture));
             }
             return sb.ToString();
         }
-        
+
         [Fact]
         [ActiveIssue(11803, TestPlatforms.AnyUnix)]
         public void Normalize()
         {
-            // Windows 8 test data came from http://www.unicode.org/Public/UCD/latest/ucd/NormalizationTest.txt 
-            // Windows 7 test came from http://www.unicode.org/Public/3.0-Update1/NormalizationTest-3.0.1.txt 
+            // Windows 8 test data came from http://www.unicode.org/Public/UCD/latest/ucd/NormalizationTest.txt
+            // Windows 7 test came from http://www.unicode.org/Public/3.0-Update1/NormalizationTest-3.0.1.txt
 
             using (Stream stream = typeof(StringNormalizationAllTests).GetTypeInfo().Assembly.GetManifestResourceStream(PlatformDetection.IsWindows7 ? "NormalizationDataWin7" : "NormalizationDataWin8"))
             {
-                Assert.True(stream != null, "Couldn't get the stream for the normalization embedded file");
+                if (stream == null)
+                {
+                    throw new ArgumentNullException("Couldn't get the stream for the normalization embedded file");
+                }
```

Comment:
Another case where I don't see the value of the change.  I'd be fine with changing it to ```Assert.NotNull(stream)```.  ArgumentNullException doesn't seem to make sense here, anyway, as it's not an argument to the function that's null, and the first parameter to ArgumentNullException is supposed to be the parameter name.